obal/data/modules/koji_whitelist_check.py

```diff
@@ -0,0 +1,33 @@
+# pylint: disable=C0111,C0301,R1710
+from ansible.module_utils.basic import AnsibleModule  # pylint: disable=C0413
+from ansible.module_utils.obal import get_whitelist_status  # pylint:disable=import-error,no-name-in-module
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.2',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+def run_module():
+    module = AnsibleModule(
+        argument_spec=dict(
+            package=dict(type='str', required=True),
+            tag=dict(type='list', required=True),
+            build_command=dict(type='str', required=True),
```

Comment:
should this be called `koji_executable` and be required=False (and then the called function defaults to `koji`)?

otherwise it's confusing that some modules require the command, and some do not.